senza/cli.py

```diff
@@ -417,11 +417,17 @@ def create(definition, region, version, parameter, disable_rollback, dry_run, fo
     for name, parameter in data.get("Parameters", {}).items():
         parameters.append([name, getattr(args, name, None)])
 
-    tags = {
+    tags = {}
+    for tag in input["SenzaInfo"].get('Tags', []):
```

Comment:
Only lists or strings are allowed in this section of the template, as they are turned into Mappings in the CF template.

    "Template format error: Every Mappings attribute must be a String or a List."

Perhaps it would be wise to allow custom tags, but in a different section of the template?